pyserini/search/__main__.py

```diff
@@ -15,13 +15,11 @@
 #
 
 import argparse
-
-from tqdm import tqdm
-
 from pyserini.search import get_topics, SimpleSearcher
 from pyserini.search.reranker import ClassifierType, PseudoRelevanceClassifierReranker
+from tqdm import tqdm
 
-parser = argparse.ArgumentParser(description='Create a input schema')
+parser = argparse.ArgumentParser(description='Perform search on a given lucene index with various options.')
```

Comment:
"Search a Lucene index" - omit needless words:
+ "perform" not necessary
+ "give"... well, of course it's given
+ "options"... well, of course there are various options